changelog/6952.removal.md

```diff
@@ -0,0 +1,3 @@
+Using the [default action](default-actions.mdx) `action_deactivate_form` to deactivate
+the currently active loop / [Form](forms.mdx) is deprecated.
```

Comment:
Mhm, I spelled with a capital `F` on purpose because it's referring to the "Rasa Forms" 🤔 Does that make sense or do you have a strong preference to use a lowercase `f`?

Reply:
makes sense, I think either way is fine 👍